src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -1153,11 +1153,11 @@ void M3(int {|#4:p5|})
                     """,
                 ExpectedDiagnostics =
                 {
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(0).WithMessage("Remove unused parameter 'p1'"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(1).WithMessage("Parameter 'p2' can be removed; its initial value is never used"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(2).WithMessage("Remove unused parameter 'p3' if it is not part of a shipped public API"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(3).WithMessage("Parameter 'p4' can be removed if it is not part of a shipped public API; its initial value is never used"),
-                    VerifyCS.Diagnostic("IDE0060").WithLocation(4).WithMessage("Parameter 'p5' can be removed; its initial value is never used"),
+                    VerifyCS.Diagnostic("IDE0060").WithSeverity(DiagnosticSeverity.Info).WithLocation(0).WithMessage("Remove unused parameter 'p1'"),
```

Comment:
💡 Use `WithArguments` instead of `WithMessage`. The message format is included in the diagnostic descriptor and does not need to be duplicated here.

Reply:
Unfortunatelly, `IDE0060` can have multiple message templates and such effect is done by enforsing message here:
https://github.com/dotnet/roslyn/blob/d84d54f2e5d3d66d81453fbd5fafbeeecd0d6cde/src/Analyzers/Core/Analyzers/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.cs#L159-L183
I would say this is a flaw in current design: here message is constucted with arguments inside, so these arguments are not considered to be arguments of diagnostic. So I tried to use `WithArguments` and it failed.

Reply:
Weird, but works for me.